wpt-schema.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
This is handy to validate whilst we're working on this, but I'm not sure it's a good idea to duplicate the schema. We could reference the actual schema, and download it when we need to check we're still conforming, but once we're up and running I don't know if we need to validate every time.

Reply:
+1 to checking it via a remote (canonical) URL/URI.

@akkomar suggested/preferred that we validate locally before sending to Telemetry, IIRC, during my call with him last night.

Reply:
(Wasn't sure from his side, now, whether it's 1) validate to schema 2) validate data 3) both, before sending?)

Reply:
The context for validating locally, in-tree (send_to_telemetry.py) was to avoid having to set up https://github.com/mozilla-services/edge-validator, IIRC.

Reply:
I think checking in a copy of the schema is a good idea, since downloading it from the web can fail. That's what we've done for perfherder, and it's saved a lot of hassle. In practice it doesn't change that often.

Reply:
Added it back.